xcube_sh/sentinelhub.py

```diff
@@ -39,6 +38,29 @@
 from .version import version
 
 
+class SentinelOAuth2Session(requests_oauthlib.OAuth2Session):
```

Comment:
The name `SentinelOAuth2Session` is not appropriate. I'll rename it to `SerializableOAuth2Session`.